torchx/test/support/torchx_case.ex

```diff
@@ -15,9 +15,17 @@ defmodule Torchx.Case do
     atol = opts[:atol] || 1.0e-4
     rtol = opts[:rtol] || 1.0e-4
 
-    assert left
-           |> Nx.all_close(right, atol: atol, rtol: rtol)
-           |> Nx.backend_transfer(Nx.BinaryBackend) ==
-             Nx.tensor(1, type: {:u, 8}, backend: Nx.BinaryBackend)
+    equals =
+      left
+      |> Nx.all_close(right, atol: atol, rtol: rtol)
+      |> Nx.backend_transfer(Nx.BinaryBackend)
+
+    if equals != Nx.tensor(1, type: {:u, 8}, backend: Nx.BinaryBackend) do
+      flunk("""
+      Tensor assertion failed.
+      left: #{inspect(left)}
+      right: #{inspect(right)}
+      """)
+    end
```

Comment:
This makes for a better error message in which we can actually see the compared tensors :)